eng/pipelines/variables/globals.yml

```diff
@@ -22,3 +22,7 @@ variables:
     value: windows
   - name: MACOS
     value: macOS
+  
```

Comment:
image.yml is meant to contain only 1es agent image information and should be separate from globals.yml

Reply:
see https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/variables/globals.yml
vs https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/variables/image.yml

Reply:
I separated the node version into a standalone file `globals.yml` and expands it in 1es-redirect.yml like we have in `image.yml`.